UIKitDZ/UIKitDZ/Food/ViewController/FoodVC.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SecondVC.swift
+//  UIKitDZ
+//
+//  Created by Ardak on 30.06.2021.
+//
+
+import UIKit
+
+final class FoodVC: UIViewController {
+    
+    //MARK: - Private property
+    
+    private let pizzaButton = UIButton(frame: CGRect(x: 74, y: 140, width: 278, height: 278))
+    private let sushiButton = UIButton(frame: CGRect(x: 74, y: 450, width: 278, height: 278))
+    
+    //MARK: - UIViewController
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupVC()
+    }
+    
+    //MARK: - Private method
+    
+    private func setupVC() {
+        self.view.backgroundColor = .systemGray3
+        navigationItem.title = "Food"
+        navigationItem.backButtonTitle = ""
+        createPizzaButton()
+        createSushiButton()
+    }
+    
+    private func createPizzaButton() {
+        pizzaButton.setImage(UIImage(named: "pizza"), for: .normal)
+        self.view.addSubview(pizzaButton)
+        pizzaButton.addTarget(self, action: #selector(nextVC), for: .touchUpInside)
+    }
+    
+    private func createSushiButton() {
+        sushiButton.setImage(UIImage(named: "sushi"), for: .normal)
+        self.view.addSubview(sushiButton)
+    }
+    //MARK: - Action
+    
+    @objc func nextVC() {
```

Comment:
```suggestion
    @objc private func nextVC() {
```
если не вызывается из вне, если так, то по расположению тоже private методы ниже должны быть, чем public / internal